tests/terminateCircularRelationships/mocks.ts

```diff
@@ -0,0 +1,31 @@
+import { A, B, C, D } from './types';
```

Comment:
This file should not be commited as it's generated by `globalSetup.ts`
You can add it to the `.gitignore` like [here](https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/main/.gitignore#L18)

And you can remove the special entry you added in `.eslintignore`

Reply:
Great!